src/SFA.DAS.AANHub.Application/Employers/Queries/GetEmployerMemberQueryValidator.cs

```diff
@@ -0,0 +1,22 @@
+﻿using FluentValidation;
+using SFA.DAS.AANHub.Domain.Interfaces.Repositories;
+
+namespace SFA.DAS.AANHub.Application.Employers.Queries
+{
+    public class GetEmployerMemberQueryValidator : AbstractValidator<GetEmployerMemberQuery>
+    {
+        public const string UserIdNotFoundMessage = "UserId was not found";
+        public GetEmployerMemberQueryValidator(IEmployersReadRepository employersReadRepository)
+        {
+            RuleFor(a => a.UserId)
+                .NotNull()
+                .NotEmpty()
+                .MustAsync(async (userId, cancellation) =>
+                {
+                    var employer = await employersReadRepository.GetEmployerByUserId(userId);
```

Comment:
Something I've found locally is that this might need to be removed if you want accurate response codes.

You're doing this check as part of the handler, but if you do it here then the response will be 400 (BadRequest) rather than 404(Not Found)